src/ratings-and-reviews/components/ReviewList.jsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import ReviewTile from './ReviewTile.jsx';
 
-const ReviewList = () => (
+const ReviewList = ({reviewList}) => (
 
   <div className="review-list">
-    <ReviewTile />
+    {reviewList.map((review, i) => <ReviewTile review={review} key={i}/>)}
```

Comment:
It might be a good idea to use something like `key={review.review_id}`. Keys are mainly useful when they're unique to the item. That way each review keeps the same key even if new ones are added or removed, which is good for performance.